build.sbt

```diff
@@ -167,7 +188,20 @@ lazy val submoduleSettings = Def.settings(
 )
 
 lazy val submoduleJvmSettings = Def.settings(
-  mimaPreviousArtifacts := Set(groupId %% moduleName.value % latestVersion),
+  mimaPreviousArtifacts := {
```

Comment:
I couldn't get `mimaReportBinaryIssues` to work without these changes. (We could potentially remove this after a release with `refined-pureconfig` has been issued.)